Tests/BSONTests/BSONCorpusTests.swift

```diff
@@ -105,7 +105,13 @@ final class BSONCorpusTests: BSONTestCase {
         let INCLUDED_CORPUS_TESTS = [
             "Int32 type",
             "Int64 type",
-            "Decimal128"
+            "Decimal128",
+            "Boolean",
+            "DateTime",
+            "Array",
+            "Document type (sub-documents)",
```

Comment:
I guess this one already worked but we just weren't including it?

Reply:
The sub-document test's need string support so we can only test this now